css/store.css

```diff
@@ -0,0 +1,181 @@
+/* Store page styling */
+main{
+  display: flex;
+}
+
+ h1{
+  color: #DE6E49;
+  width: 75%;
+  font-size: 2.7rem;
+  margin: 50px auto;
+}
+
+section{
+  width: 55%;
+  height: 770px;
+ 
+}
+
+aside{
+  width: 43%;
+  background-image: url(/level-2/store-image_by-andrew-neel-unsplash.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+aside>img{
+  width: 100%;
+}
+
+form{
+  display: grid;
```

Comment:
What's this display grid doing? I think it's extra. Overall you code is so clean and very easy to read! 😍